### PR TITLE
hwdb: Add override for X280 trackpoint

### PR DESCRIPTION
**Pull request by benzea**
The trackpoint is really sensitive by default. Lowering the sensitivity
makes it much better to control.

Device in question is:

P: /devices/rmi4-00/rmi4-00.fn03/serio3/input/input20
E: DEVPATH=/devices/rmi4-00/rmi4-00.fn03/serio3/input/input20
E: EV=7
E: ID_BUS=rmi
E: ID_INPUT=1
E: ID_INPUT_MOUSE=1
E: ID_INPUT_POINTINGSTICK=1
E: KEY=70000 0 0 0 0
E: MODALIAS=input:b0011v0002p000Ae0000-e0,1,2,k110,111,112,r0,1,amlsfw
E: NAME="TPPS/2 ALPS TrackPoint"
E: PHYS="synaptics-rmi4-pt/serio1/input0"
E: PRODUCT=11/2/a/0
E: PROP=21
E: REL=3
E: SUBSYSTEM=input
E: TAGS=:seat:
E: USEC_INITIALIZED=17726885

P: /devices/rmi4-00/rmi4-00.fn03/serio3/input/input20/event15
N: input/event15
E: DEVNAME=/dev/input/event15
E: DEVPATH=/devices/rmi4-00/rmi4-00.fn03/serio3/input/input20/event15
E: ID_BUS=rmi
E: ID_INPUT=1
E: ID_INPUT_MOUSE=1
E: ID_INPUT_POINTINGSTICK=1
E: LIBINPUT_DEVICE_GROUP=11/2/a:synaptics-rmi4-pt/serio1
E: MAJOR=13
E: MINOR=79
E: POINTINGSTICK_SENSITIVITY=64
E: SUBSYSTEM=input
E: USEC_INITIALIZED=17762957

### CODE REVIEW

File: hwdb/70-pointingstick.hwdb [hunk 1]
```diff
@@ -142,3 +142,7 @@ evdev:name:TPPS/2 IBM TrackPoint:dmi:bvn*:bvr*:bd*:svnLENOVO:pn*:pvrThinkPadX20?
 evdev:input:b0003v06CBp0009*
  POINTINGSTICK_CONST_ACCEL=2.5
  POINTINGSTICK_SENSITIVITY=200
+
+# ThinkPad X280 with an ALPS trackpoint
+evdev:name:*ALPS TrackPoint*:svnLENOVO:*:pvrThinkPadX280:*
+ POINTINGSTICK_SENSITIVITY=64
```
Comment (hadess) [inline hwdb/70-pointingstick.hwdb:148]: As mentioned on IRC, the header says `"Entries should be sorted with growing _SENSITIVITY and _CONST_ACCEL."` so this should be at the top of the section.

**Comment by hadess** [comment]
CC @whot 

I was looking at whether I could push this quickly, but I really wasn't sure about the number, given that the methodology seems to be "try numbers until it feels right" which doesn't seem very scientific or reproducible. If there is a better way to get those numbers, could you add them to the header (or add a reference where this is explained in the header)?

**Comment by benzea** [comment]
So, the number is applied, but libinput also adjusts things accordingly so one can only see the improved behaviour between triggering udev and libinput initialising again.

Moved to: https://bugs.freedesktop.org/show_bug.cgi?id=105485

**Comment by whot** [comment]
There is no reproducible way to measure this, short of building a robot that can measure pressure-force. The only thing we can measure is *maximum* pressure and then guess that a only a percentage of that actually matters.

Generally the `POINTINGSTICK_*` properties should not be set anymore, libinput doesn't use it and now works around it, effectively by undoing the effect of the `SENSITIVITY` property. As much as it can anyway... 

We still have backwards compatibility with it but new devices should use the trackpoint range instead. Except that doesn't appear to work well for all devices but it's hard to get good data. It's chaos right now, hence the complete switch yet to remove things from the systemd hwdb hasn't happened.

TLDR: skip this one, use the trackpoint range in libinput instead
